exporters/etw/include/opentelemetry/exporters/etw/etw_provider.h

```diff
@@ -203,7 +203,7 @@ class ETWProvider
   {
     std::lock_guard<std::mutex> lock(m_providerMapLock);
 
-    auto m  = providers();
+    auto &m = providers();
```

Comment:
Add a comment? Or could we disallow `providers()` get copied?

Reply:
Thanks, added a comment.